src/Dotenv.php

```diff
@@ -35,8 +36,6 @@ public function __construct($path, $file = '.env')
      */
     public function load()
     {
-        $this->loader = new Loader($this->filePath, $immutable = true);
```

Comment:
So, if i'm getting this right, You would have the line in the constructor and in the `load` method, and the load method would overwrite the "default" loader with a new one?